### DISCUSSION TITLE
Problem with Pico W

### DISCUSSION THREAD

**Discussion by ramon2k10** [fhdm-dev/scoppy#147] [Q&A]
Hi, I flashed my Pico W with the scoppy-picow-v18.uf2 version. The green led only blink once and stop to blink. I can see a new WiFi access point called ESP_DB38FA but never see something like SCOPPY_xxxxx. I can't do any progress. I don't now what I'm doing wrong. This version neither can connect using USB OTG. 

In the other hand when I flash the PICO version everything goes well using USB OTG

Can anybody help me?

Regards

**Comment by fhdm-dev** [comment]
Hi,

Is it possible you have a clone Pico W board and not a genuine Pico W? The Scoppy firmware only works with a genuine Pico W board.

**Reply by sjgcornish** [reply]
I have a genuine RPI pico W. My Andoid phone see the SCOPPY_xxxxx. network and connects to it. The scoppy app shows trying to connect at 192.168.4.1 bur will not connect to it. I cannot ping 192.168.4.1 from the phone. Running pico from seperate power supply and alos numed it and reinstalled the scoppy app. Note led flashes  as the instructions indicate. Any ideas. Could the phone be precenting connecting to an unsequired network.

**Reply by fhdm-dev** [reply]
Some phones will automatically disconnect from an access point if they detect that there is no internet access. Could this be happening?

**Reply by sjgcornish** [reply]
Thank you. You are right I had to put the phoe in Flight mode to stop itt switching to mobile network when wifi sees no internet connection
